src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #{$color-white};
+  border: 1px solid #{$main-border-color};
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+
```

Comment:
don't add many unnecessary blank lines

Reply:
it' s called css code style , to make it more readable)
Check this)
https://codeguide.academy/html-css.html#css-order